### PR TITLE
More v4 Fixes

### PR DESCRIPTION
**Pull request by binaryoverload**
This pull request resolves #XXX

 - [x] I have read and agreed to the [code of conduct](https://github.com/movie-web/movie-web/blob/dev/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/movie-web/movie-web/blob/dev/.github/CONTRIBUTING.md).
 - [ ] What I'm implementing was assigned to me and is an [approved issue](https://github.com/movie-web/movie-web/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved). For reference, please take a look at our [GitHub projects](https://github.com/movie-web/movie-web/projects).
 - [x] I have tested all of my changes.

### CODE REVIEW

**Comment by binaryoverload** [comment]
Dropdown looks like:

image

**Comment by binaryoverload** [comment]
Sticky dropdown:

Small screens:
image

Bigger screens:
image

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "node-forge": "^1.3.1",
     "ofetch": "^1.0.0",
     "react": "^17.0.2",
+    "type-fest": "^4.3.3",
```
Comment (mrjvs) [inline package.json:30]: why has this been moved to being a runtime dependency?

File: src/assets/locales/en.json [hunk 2]
```diff
@@ -398,7 +399,7 @@
     }
   },
   "footer": {
-    "tagline": "Watch your favorite shows and movies with this open source streaming app.",
+    "tagline": "Watch your favourite shows and movies with this open source streaming app.",
```
Comment (mrjvs) [inline src/assets/locales/en.json:402]: lol

**Review by mrjvs — CHANGES_REQUESTED** [review]
linting has also thrown some warnings

File: package.json (hunk 1 above)
Comment (binaryoverload) [reply package.json:30]: EsLint complained about the import, although in retrospect you can do `import type` and it wouldn't complain

**Review by mrjvs — APPROVED** [review]

**Pull request merged**